vmdb/app/models/ems_kubernetes.rb

```diff
@@ -20,8 +20,16 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
-  def self.raw_connect(hostname, port, _service = nil)
-    kubernetes_connect(hostname, port)
+  def supported_auth_types
+    %w(basic)
+  end
+
+  def supports_authentication?(authtype)
```

Comment:
> ems_kubernetes.rb, ems_openstack.rb, ems_redhat.rb, ext_management_system.rb all are using the same "authtype" (and a really similar function)

I think that's an unclear name, and the fact other places in code (which were written a while ago afaik) use it doesn't mean we should too :) 
I'm open to hear other folks opinions, but needless to say - don't let this nit block you.